.codeclimate/checkstyle.xml

```diff
@@ -109,9 +109,6 @@
 
         <!-- Checks for Size Violations.                    -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
-        <module name="LineLength">
-            <property name="max" value="150"/>
-        </module>
```

Comment:
Okay so I have no idea why this is still erroring. But perhaps we can ignore the codeclimate issue for now and see why it is complaining about this module being a child of `TreeWalker` while it clearly isn't (and is even deleted). @julian9499 I would move this one to be a direct child of `Checker` as per documentation, and then just merge this PR for now and add an issue to look into this. Perhaps codeclimate is just being weird and not using the correct config file, but using the one on the master branch for some stupid reason. Maybe that fixes it, and otherwise we'll have the issue to remind us to fix it. No need to keep the 3 PRs open for now.

Reply:
Yeah that might be better